src/components/footer/Footer.css

```diff
@@ -43,11 +43,13 @@ footer {
     width: 60px;
     height: 60px;
     padding-right: 0.4rem;
+    border-radius: 0;
 }
 
 .footer-text-logo {
     width: 48px;
     height: 48px;
+    border-radius: 0;
```

Comment:
First of all, this change is not related to the PR.
Secondly, the fix should be in InterviewPage.css. Need to remove the corder-radius from img and move it to interview-profile.

Reply:
I removed and will do it in next PR